WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml

```diff
@@ -0,0 +1,196 @@
+                    <ImageSource x:Key="CardsImage">ms-appx:///Assets/Design/Cards.dark.png</ImageSource>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="Light">
+                    <ImageSource x:Key="DialogImage">ms-appx:///Assets/Design/Dialog.light.png</ImageSource>
+                    <ImageSource x:Key="CardsImage">ms-appx:///Assets/Design/Cards.light.png</ImageSource>
+                </ResourceDictionary>
+            </ResourceDictionary.ThemeDictionaries>
+
+
+            <Style x:Key="GridCardStyle" TargetType="Grid">
+                <Style.Setters>
+                    <Setter Property="HorizontalAlignment" Value="Stretch" />
+                    <Setter Property="MinHeight" Value="68" />
+                    <Setter Property="CornerRadius" Value="{StaticResource OverlayCornerRadius}" />
+                </Style.Setters>
+            </Style>
+
+            <Style x:Key="ValueTextStyle" TargetType="TextBlock">
+                <Style.Setters>
+                    <Setter Property="Margin" Value="16,0,0,0" />
+                    <Setter Property="VerticalAlignment" Value="Center" />
+                </Style.Setters>
+            </Style>
+
+            <Style
+                x:Key="DescriptionTextStyle"
+                BasedOn="{StaticResource CaptionTextBlockStyle}"
+                TargetType="TextBlock">
+                <Style.Setters>
+                    <Setter Property="Grid.Column" Value="2" />
+                    <Setter Property="TextWrapping" Value="Wrap" />
+                    <Setter Property="VerticalAlignment" Value="Center" />
+                </Style.Setters>
+            </Style>
+
+            <Style x:Key="VisualBorderStyle" TargetType="Border">
+                <Style.Setters>
+                    <Setter Property="Grid.Column" Value="1" />
+                    <Setter Property="CornerRadius" Value="{StaticResource ControlCornerRadius}" />
+                    <Setter Property="Height" Value="20" />
+                    <Setter Property="HorizontalAlignment" Value="Left" />
+                    <Setter Property="Background" Value="{ThemeResource AccentFillColorDefaultBrush}" />
+                </Style.Setters>
+            </Style>
+
+        </ResourceDictionary>
+    </Page.Resources>
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <RichTextBlock>
+            <Paragraph>
+                <Run>The use of consistently sized spacing and gutters semantically groups an experience into separate components. These values map to our rounded corner logic and together help create a cohesive and usable layout.</Run>
+                <Run>A best practice in design is to use a 4px grid. This means that any spacing or sizing should be a multiple of 4. This helps to create a consistent and harmonious layout.</Run>
```

Comment:
... And makes for nice values when scaling

Reply:
Added